lib/xcpretty/parser.rb

```diff
@@ -193,7 +193,7 @@ module Warnings
 
       # @regex Captured groups
       # $1 = whole warning
-      GENERIC_WARNING_MATCHER = /^warning:\s(.*)$/
```

Comment:
1. - it can be very significant, the most extreme case i've seen is from a "known company"'s build log that took ~40 minutes for `xcpretty` to spit it out.

2. As we're switching from whitelisting to [blacklisting](https://github.com/supermarin/xcpretty/pull/118), this will not be the issue anymore.